io.github.JaredTweed.AudioToTextTranscriber.yml

```diff
@@ -0,0 +1,52 @@
+app-id: io.github.JaredTweed.AudioToTextTranscriber
+runtime: org.gnome.Platform
+runtime-version: "48"
+sdk: org.gnome.Sdk
+
+command: audio-to-text-transcriber
+
+finish-args:
+  - --share=network
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --device=dri
+  - --filesystem=home
+
+modules:
+  - name: whisper-and-py-src
+    buildsystem: cmake-ninja 
+    sources:
+      - type: archive
+        url: https://github.com/JaredTweed/Audio-To-Text-Transcriber/releases/download/v1.0.0/v1.0.0.zip
+        sha256: 5def3e7bf74b9729259f6071664245c2dfd316c0ada967fb11d9d5785db8722f
+        strip-components: 1 
+    # let the cmake-ninja helper do the cmake + ninja work
+    config-opts:
+      - -DCMAKE_BUILD_TYPE=Release
+      - -DCMAKE_INSTALL_PREFIX=/app
+      - -DCMAKE_INSTALL_RPATH=/app/lib
+
+    # install the Python GUI and whisper.cpp source tree
+    post-install:
+      - install -d /app/share/audio-to-text-transcriber
+      - cp -a . /app/share/audio-to-text-transcriber/
+
+  - name: metadata
+    buildsystem: simple
+    sources:
+      # instead of `path: .`, explicitly fetch each file from the release ZIP
+      - type: archive
+        url: https://github.com/JaredTweed/Audio-To-Text-Transcriber/releases/download/v1.0.0/v1.0.0.zip
```

Comment:
this is the same tarball as above but you ignored and resolved my comment https://github.com/flathub/flathub/pull/6442/files#r2067676891

Please follow the reviewer requests.

Reply:
it's even worse, it's the same URL but a different checksum. One of them is incorrect.